aws/templates/docker_compose_xchain.tftpl

```diff
@@ -18,7 +18,7 @@ services:
             - --config-file=/config/${xchain_config}.json
             - --omni-rpc=${omni_rpc}
 %{ endif ~}
-            image: omniops/xchain-indexer:latest
+            image: ${xchain_docker_image}
```

Comment:
> xchain_indexer_docker_image

Renamed. The var is defined in the docker_compose_config vars as part of module "ec2_asg_xchain_indexer" and module "ec2_asg_xchain_api".

Reply:
> also – note that the `xchain-api` container in the "if branch" also needs to use the updated config. Since it uses the same docker image

Yep, will migrate it as well once we are done with testing the current changes..